Gem/Code/Source/AutoGen/NetworkAiComponent.AutoComponent.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+
+<Component
+    Name="NetworkAiComponent" 
+    Namespace="MultiplayerSample" 
+    OverrideComponent="true" 
+    OverrideController="false" 
+    OverrideInclude="Source/Components/NetworkAiComponent.h"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+    <NetworkProperty Type="bool" Name="Enabled" Init="false" ReplicateFrom="Authority" ReplicateTo="Client" Container="Object" IsPublic="true" IsRewindable="false" IsPredictable="false" ExposeToEditor="true" ExposeToScript="false" GenerateEventBindings="false" Description="If enabled, this AI component overrides movement and camera components." />
+    <NetworkProperty Type="uint32_t" Name="OwningConnectionId" Init="0xffffffff" ReplicateFrom="Authority" ReplicateTo="Client" Container="Object" IsPublic="true" IsRewindable="false" IsPredictable="false" ExposeToEditor="false" ExposeToScript="false" GenerateEventBindings="false" Description="Defines whether the host or a particular client should drive the AI of this component's parent entity. A max uint32_t value indicates that the server ticks the AI."/>
```

Comment:
OwningConnectionId going over the wire has me on edge. I will read on to see how this is used!

Reply:
Looks like it's not used by anything on the other side receiving replicated updates for the entity. In fact it doesn't really appear to be used at all. This is good, since the value would be meaningless to anyone but the original host (connectionId is specific to the INetworkInstance that owns it). If you really need this to be a network property, I might change it to Authority to Authority. Otherwise I might just make this a regular member variable on your derived component/controller or remove it entirely.

Reply:
Will remove and circle back offline to understand how to execute this pattern